examples/components/MNIST/traininsilo/run.py

```diff
@@ -32,6 +42,11 @@ def __init__(
             lr (float, optional): Learning rate. Defaults to 0.01
             epochs (int, optional): Epochs. Defaults to 1
             batch_size (int, optional): DataLoader batch size. Defaults to 64
+            dp (bool, optional): Differential Privacy. Default is False (Note: dp, dp_target_epsilon, dp_target_delta, dp_max_grad_norm, and total_num_of_iterations are defined for the only purpose of DP and can be ignored when users don't want to use Differential Privacy)
```

Comment:
For "Default is False", I saw capital "False" here, but in the spec.yaml file, config.yaml file as well as quickstart.md, I saw lower case "false/true" there. Better to verify and also be consistent to avoid any potential pitfalls. (generally Boolean values need to use capitalized "True/False" but for special contexts "true/false" might also possibly work fine, but better to verify and be consistent.)

Reply:
I intentionally did that to be consistent with other files bcoz in all our yaml/md file, the boolean is defined with lowercase t/f ..For instance, [azure-ml-federated-learning/spec.yaml at release-05 · Azure-Samples/azure-ml-federated-learning (github.com)](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/release-05/examples/components/HELLOWORLD/traininsilo/spec.yaml#L21).